app/services/service_sign_in_publish_service.rb

```diff
@@ -3,6 +3,6 @@ def self.call(content)
     content_id = content.content_id
     Services.publishing_api.put_content(content_id, content.render_for_publishing_api)
     Services.publishing_api.patch_links(content_id, links: content.links)
-    Services.publishing_api.publish(content_id)
+    Services.publishing_api.publish(content_id, content.update_type, locale: content.locale)
```

Comment:
We don't need to send `update_type` in publish - it has been deprecated in favour of sending it in the put_content payload (see https://github.com/alphagov/publishing-api/blob/master/doc/api.md#post-v2contentcontent_idpublish)

Reply:
Sure, but it's an optional argument which is only there because lots of apps are still sending it with publish when they don't need to (see https://github.com/alphagov/gds-api-adapters/pull/719).  If we included it now we'd only have to remove it in future, so I think we shouldn't include this in our code now.